configure

```diff
@@ -857,12 +842,13 @@ def configure_intl(o):
     # ICU from pkg-config.
     o['variables']['v8_enable_i18n_support'] = 1
     pkgicu = pkg_config('icu-i18n')
-    if not pkgicu:
+    if pkgicu[0] == None:
       print 'Error: could not load pkg-config data for "icu-i18n".'
       print 'See above errors or the README.md.'
       sys.exit(1)
-    (libs, cflags) = pkgicu
+    (libs, cflags, libpath) = pkgicu
     o['libraries'] += libs.split()
+    o['libraries'] += libpath.split()
```

Comment:
Will do, this is wrong though -- libpath needs to be treated as a string similar to pkg-config. I might look at refactoring this to using `configure_library`  instead.